modules/sno-du-removing-the-console-operator.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-removing-the-console-operator_{context}"]
+= Removing the console Operator
```

Comment:
Should be Disabling the console operator

Reply:
changed